conjur/logic/credential_provider/keystore_credentials_provider.py

```diff
@@ -27,21 +28,21 @@ class KeystoreCredentialsProvider(CredentialsStoreInterface):
     in the system's keystore
     """
 
-    def __init__(self): # pragma: no cover
+    def __init__(self):  # pragma: no cover
         pass
 
-    # TODO assign variable identifier = credential_data.machine and use it inside set_password
     # pylint: disable=line-too-long,logging-fstring-interpolation
     def save(self, credential_data: CredentialsData):
         """
         Method for saving user credentials in the system's keyring
         """
         logging.debug("Attempting to save credentials to the system's credential store "
                       f"'{KeystoreAdapter.get_keyring_name()}'...")
+        cred_id = credential_data.machine
```

Comment:
Maybe it is a preference but I always like full, wholesome names
```suggestion
        credential_id = credential_data.machine
```

Reply:
done